src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier_transports.xml

```diff
@@ -26,6 +26,10 @@
             <tag name="texter.transport_factory" />
         </service>
 
+        <service id="notifier.transport_factory.ovhcloud" class="Symfony\Component\Notifier\Bridge\OvhCloud\OvhCloudTransportFactory" parent="notifier.transport_factory.abstract">
+            <tag name="texter.transport_factory" />
+        </service>
```

Comment:
You also need to update `src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php`, see #33875 for an example.